### PR TITLE
Allow Custom OPENAPS_DIR

### PR DESCRIPTION
**Pull request by efidoman**

### CODE REVIEW

File: bin/oref0-log-shortcuts.sh [hunk 1]
```diff
@@ -1,5 +1,8 @@
 #!/bin/bash
 
+[ -z "$OPENAPS_DIR" ] && OPENAPS_DIR="~/myopenaps"
+myopenaps="$OPENAPS_DIR"
```
Comment (alimhassam) [inline bin/oref0-log-shortcuts.sh]: minor, but this could be simplified to 
```
myopenaps=${OPENAPS_DIR:-"~/myopnaps"}
```

**Comment by efidoman** [comment]
Simplified environment variable check per @alimhassam suggestion.

File: bin/oref0-log-shortcuts.sh [hunk 2]
```diff
@@ -9,18 +11,18 @@ grep -q autotunelog ~/.bash_profile || echo "alias autotunelog="'"tail -n 100 -F
...
 grep -q cat-wifi ~/.bash_profile || echo "alias cat-wifi="'"cat /etc/wpa_supplicant/wpa_supplicant.conf"' >> ~/.bash_profile
 grep -q edit-wifi ~/.bash_profile || echo "alias edit-wifi="'"nano /etc/wpa_supplicant/wpa_supplicant.conf"' >> ~/.bash_profile
-grep -q cat-runagain ~/.bash_profile || echo "alias cat-runagain="'"cd ~/myopenaps && cat oref0-runagain.sh"' >> ~/.bash_profile
-grep -q edit-runagain ~/.bash_profile || echo "alias edit-runagain="'"cd ~/myopenaps && nano oref0-runagain.sh"' >> ~/.bash_profile
-grep -q cat-autotune ~/.bash_profile || echo "alias cat-autotune="'"cd ~/myopenaps/autotune && cat autotune_recommendations.log"' >> ~/.bash_profile
+grep -q cat-runagain ~/.bash_profile || echo "alias cat-runagain=\"cd $myopenaps && cat oref0-runagain.sh\"" >> ~/.bash_profile
+grep -q edit-runagain ~/.bash_profile || echo "alias edit-runagain=\"cd $myopenaps && nano oref0-runagain.sh\"" >> ~/.bash_profile
+grep -q cat-autotune ~/.bash_profile || echo "alias cat-autotune=\"cd ${myopenaps}/autotune && cat autotune_recommendations.log\"" >> ~/.bash_profile
```
Comment (scottleibrand) [inline bin/oref0-log-shortcuts.sh]: Why curly braces here but not the prior ones?  We should probably make them all consistent.

**Comment by efidoman** [comment]
@scottleibrand - changed to all curly brackets per your suggestion.

File: www/templates/index.html [hunk 3]
```diff
@@ -127,7 +127,7 @@
                         },
                         options: {
                             responsive: true,
-                            legend: {
+     >>>>>>> dev                       legend: {
```
Comment (scottleibrand) [inline www/templates/index.html]: Looks like you introduced three merge conflict artifacts here that need to be removed.

**Pull request merged**